urlwatch

```diff
@@ -243,19 +252,30 @@ if __name__ == '__main__':
             os.makedirs(needed_dir)
 
     # Check for required files
-    if not os.path.isfile(urls_txt):
-        log.warning('not a file: %s' % urls_txt)
-        urls_txt_fn = os.path.join(os.path.dirname(urls_txt), os.path.basename(urls_txt_example))
+    if not os.path.isfile(urls_yaml):
+        log.warning('not a file: %s' % urls_yaml)
+        urls_yaml_fn = os.path.join(os.path.dirname(urls_yaml), os.path.basename(urls_yaml_example))
         hooks_py_fn = os.path.join(os.path.dirname(hooks_py), os.path.basename(hooks_py_example))
-        print ERROR_MESSAGE_URLS_TXT % (urls_txt, urls_txt_fn)
+        if os.path.isfile(urls_txt):
+            # import old urls.txt file
+            handler.create_urls_yaml(handler.parse_urls_txt(urls_txt), urls_yaml)
+            # rename urls.txt after import
+            rename(urls_txt, "%s.imported" % urls_txt)
+            print IMPORT_MESSAGE_URLS % (urls_yaml, urls_yaml_fn)
+        else:
+            print ERROR_MESSAGE_URLS % (urls_yaml, urls_yaml_fn)
+
         if not options.hooks:
             print ERROR_MESSAGE_HOOKS_PY % (hooks_py, hooks_py_fn)
-        if os.path.exists(urls_txt_example) and not os.path.exists(urls_txt_fn):
-            shutil.copy(urls_txt_example, urls_txt_fn)
+        if os.path.exists(urls_yaml_example) and not os.path.exists(urls_yaml_fn):
+            shutil.copy(urls_yaml_example, urls_yaml_fn)
         if not options.hooks and os.path.exists(hooks_py_example) and not os.path.exists(hooks_py_fn):
             shutil.copy(hooks_py_example, hooks_py_fn)
         sys.exit(1)
 
+    if os.path.isfile(urls_txt):
+        log.warning('Old urls.txt found, but ignored.')
```

Comment:
Maybe log as:

    log.warning('Ignoring old urls.txt file at %s', urls_txt)

This way, in case the user is unsure where the program found the file, they have an easier time removing it.